Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -195,6 +191,8 @@
     NoSessions = 1;
 end
 
+nSessions = numel(SessionList_qCBF); % define number of sessions

```

Comment:
nSessions can be left here only and removed from line 190

Reply:
I think it needs to stay, otherwise nSessions doesnt get defined because `` nSessions = numel(SessionList_qCBF) `` would be set to 0 and is required for the FOR loop later on. See:

![image](https://user-images.githubusercontent.com/75380250/114877342-79c35d80-9dff-11eb-969d-9e189011906b.png)

Reply:
Sorry - that was confusing. I can only enter comments on the lines you have modified and not on other lines. This line 194 is OK and it indeed has to stay it this way. But a few lines above - line 190, the nSessions is never used as it is overwritten on line 194 in all cases.

Reply:
Okay I'll change it!

Reply:
I've change it to make it a bit more robust (and testing now):

![image](https://user-images.githubusercontent.com/75380250/115041567-efe4c480-9ed2-11eb-98a5-f28e3331074a.png)

Otherwise SessionsList remains empty and using ``cellfun`` and ``numel`` might give an error.

Reply:
Running into a new problem

![image](https://user-images.githubusercontent.com/75380250/115246697-69282580-a126-11eb-80dc-f9bb9e475c68.png)

Reply:
Okay multiple steps are going wrong. To inventarise: 
- Defining sessions by looking at qCBF.nii's is only possible if we differentiate between subject number and sessions number
- Length of x.S.SUBJECTID goes wrong somewhere